wiki/Article_Style_Guide/en.md

```diff
@@ -3,28 +3,74 @@
 The Article Style Guide serves as a way to help keep consistency in clarity, formatting, and layout between all articles of the osu!wiki.
 
 All English articles should use plain English.
-Your word choice should be able to explain such things in layman's terms (basically ask yourself, "If the reader is reading this, would they have to lookup any of the words?").
+Your word choice should be able to explain such things in layman's terms.
+In other words, you should ask yourself, "If the reader is reading this, would they have to lookup any of the words?"
 
 All articles must have proper grammar, correct spelling, and correct information.
-Know that reviewers may ask for changes in your pull request for blunders or suggestions.
-A good osu!wiki writer should read these reviews to help improve the overall quality of these articles to ensure optimal experience for an osu!wiki reader.
+Keep in mind that reviewers may ask for changes in your pull request for blunders or suggestions.
+A good osu!wiki writer/editor should read these reviews to help improve the overall quality of these articles to ensure optimal experience for the reader.
 
 ## GFM
 
 GFM (GitHub Flavored Markdown) is the supported markup language used in the osu!wiki.
-HTML use is **heavily** discouraged (and might have your pull request be denied)!!
+HTML use is **heavily** discouraged (and might have your pull request be denied)!
 
-GFM has a small amount of markup that **anyone** can learn it in less than a day!
-However, don't expect a lot out of Markdown as it is intended to be used as a bare minimum markup language.
+GFM has a small amount of markup that **anyone** can learn it in less than a minute!
```

Comment:
according to [GitHub GFM guide](https://guides.github.com/features/mastering-markdown/) it probably use 3 minutes